tests/suite/test_virtual_server_grpc.py

```diff
@@ -63,7 +63,6 @@ def fin():
 
 @pytest.mark.vs
 @pytest.mark.smoke
-@pytest.mark.ciara
```

Comment:
haha I do that all the time

Reply:
I usually put "@pytest.mark.sean"

Reply:
@soneillf5 has a script to detect it now